src/app-root.ts

```diff
@@ -558,6 +575,26 @@ export class AppRoot extends LitElement {
     }
   }
 
+  private resultLastTileViewCheckboxChanged(e: Event) {
+    const target = e.target as HTMLInputElement;
+
+    const div = document.createElement('div');
+    const title = document.createElement('h3');
+    title.textContent = 'Upload';
+
+    div.setAttribute('slot', 'result-last-tile');
+    div.setAttribute('class', 'result-last-tile');
+    div.appendChild(title);
```

Comment:
Q: We still construct the new element even when the box is unchecked -- could we avoid this?

Reply:
I don't think is construct without clicking on the checkbox. I verified with some console statement. Thannks!